spec/redux/reducer-test.js

```diff
@@ -0,0 +1,36 @@
+import loadApi from '../../src/redux/reducer';
+import * as actType from '../../src/redux/action-types';
+
+describe("The reducer", () => {
+
+  it("returns a state object", () => {
+    expect(loadApi({}, "default")).toEqual(jasmine.any(Object));
```

Comment:
You can pass in `undefined` for the state to test out the initialization behavior.